packages/terra-framework-docs/src/terra-dev-site/doc/worklist-data-grid/Examples.2/RowSelection.jsx

```diff
@@ -146,6 +276,7 @@ const RowSelection = () => {
           disableSelectableRows();
```

Comment:
We would want our consumers to pass anonymous functions to us.  It will cause unnecessary rerenders.

Reply:
Updated